CHANGELOG.md

```diff
@@ -298,13 +298,17 @@ You can now add attributes like classes, rowspan and colspan to table row header
 
 [Pull request #1367: Allow for classes, rowspan, colspan and attributes on row headers](https://github.com/alphagov/govuk-frontend/pull/1367). Thanks to [edwardhorsford](https://github.com/edwardhorsford).
 
-### Use page wrappers without a modifier
+### Use page wrapper auto spacing
 
-If your page wrapper does not have components above the main section, you no longer need to use the [`govuk-main-wrapper--l` modifier class](https://design-system.service.gov.uk/styles/layout/#exploded-view-of-page-wrappers-without-a-back-link-breadcrumbs-or-phase-banner) to increase the vertical space above the content.
+You can now use the the `.govuk-main-wrapper--auto-spacing` modifier on the main wrapper.
```

Comment:
Tweaked this to match the language we use on https://design-system.service.gov.uk/styles/layout/.

```suggestion
You can now use the `.govuk-main-wrapper--auto-spacing` modifier class in your `<main>` element to add responsive padding to the top and bottom of the page.
```